### DISCUSSION TITLE
2024/08/19/finiteness-in-sheaf-topoi

### DISCUSSION THREAD

**Discussion by giscus** [HallaSurvivor/Blog#30] [Announcements]
# 2024/08/19/finiteness-in-sheaf-topoi

Chris Grossack's math blog and professional website.

https://grossack.site/2024/08/19/finiteness-in-sheaf-topoi.html

**Comment by 0rangeM** [comment]
Hi Chris! Another very nice post making topos theory accessible to more people (probably saving me a lot of work in the future, thanks for that). You mentioned Sketches of an Elephant in a footnote, but something that continues to mystify me is the unqualified use of "finite" in earlier sections of the book (when giving "elementary" definitions). I'm sure I'll figure out how to navigate that eventually.

Quick note:

> For instance, we can see the fact that cardinals always have decidable equality by noticing that any two sections over $U$ are either equal on all of $U$ or none of $U$!

Just before this, you gave an example of a space with two connected components where two sections could be equal on just part of $U$. It's still true that we have decidability, but the explanation is more that any $U$ is covered by the subspace on which the sections are equal and the subspace on which they are not equal (which is more clearly the interpretation in your counterexample immediately afterwards and the note later on). Maybe you intended for $U$ to be an open in $[-1,1]$ specifically? Or a connected open?

**Reply by HallaSurvivor** [reply]
Ah, good catch! I thought I included a footnote saying that this is true only when $U$ is connected, but since truth is local we can always pass to connected components of $U$. I must have either deleted that footnote or only _thought_ I wrote it, or got it mixed up with footnote 8... Regardless, I think the point is clear! 

As for Johnstone, I _think_ that whenever he says "finite" he means "bishop finite". But I haven't gone through and checked this carefully.
